src/value.cpp

```diff
@@ -993,6 +975,7 @@ namespace Exiv2 {
 
     std::ostream& DateValue::write(std::ostream& os) const
     {
+        // Write DateValue in ISO 8601 Extended format: YYYY-MM-DD
```

Comment:
If so, then missing padding w/ 0 to width 4 for they date_.year below?

Reply:
Good catch. I will add the padding although as it is mentioned in the ISO:

> Values in the range [0000] through [1582] shall only be used by mutual agreement of
the partners in information interchange.

Note that I did not change that code. Probably most of the dates handled by Exiv2 will be between 19xx & 20XX